metadata/command.xml

```diff
@@ -45,5 +45,18 @@
 				<_long>Sets the triggering activator binding.</_long>
 			</entry>
 		</option>
+		<option name="release_bindings" type="dynamic-list">
```

Comment:
Since this PR was created, we also had some changes to the dynamic-list options. They should have a `type-hint` property and entries should have `names`. Would you mind adding them here? (The other command options have been updated, so you can use them as a reference)

Reply:
I have fixed the xml attributes. Thanks for all the help!